Issues seem(?) to only trigger when gcc13 is combined with musl.

https://bugs.gentoo.org/906363
--- a/modules/fbx/fbx_parser/FBXCommon.h
+++ b/modules/fbx/fbx_parser/FBXCommon.h
@@ -78,2 +78,3 @@
 
+#include <cstdint>
 #include <string>
--- a/thirdparty/vhacd/inc/vhacdManifoldMesh.h
+++ b/thirdparty/vhacd/inc/vhacdManifoldMesh.h
@@ -17,2 +17,3 @@
 #define VHACD_MANIFOLD_MESH_H
+#include <cstdint>
 #include "vhacdCircularList.h"
